app/controllers/stops_controller.rb

```diff
@@ -7,4 +7,20 @@ def show
     @stop=Stop.find(params[:id])
   end
 
+  def create
+    new_stop = Stop.new(params[:stop])
+
+    if new_stop.save
+      flash[:notice] = "stop created successfully"
+
+    else
+      flash[:error] = "error"
+
```

Comment:
😭you guys are killing me with your extra lines, but I'll let it go!

Reply:
sorry about that. that file accidentally got included when we switched branches. I'll be sure to throw in a few extra lines for you next time though :)